src/test/java/gov/healthit/chpl/aqa/stepDefinitions/FuzzyMatchTextSteps.java

```diff
@@ -25,7 +25,7 @@ public FuzzyMatchTextSteps() {
 
     @Given("^I am on listing details page \"([^\"]*)\"$")
     public void iAmOnListingDetailsPage(final String arg1) throws Throwable {
-        driver.get("https://chpl.ahrqstg.org/#/product/" + arg1);
+        driver.get("https://chpl.ahrqdev.org/#/product/" + arg1);
```

Comment:
Please update to use the URL stuff, or get rid of this step entirely and use the other one that does the same thing